client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/composables/table/BpTable.kt

```diff
@@ -31,6 +31,7 @@ data class Header(val text: String, val weight: Float = 1f)
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun <T> ColumnScope.BpTable(
+    hasConnection: Boolean = true,
```

Comment:
just the arrangement of parameters , parameters that hasn't default value recommended to be first